tethys_cli/manage_commands.py

```diff
@@ -35,10 +41,13 @@ def add_manage_parser(subparsers):
     manage_parser.add_argument('-l', '--link', required=False, action='store_true',
                                help='Only used with collectstatic command. Link static directory to STATIC_ROOT '
                                     'instead of copying it. Not recommended.')
-    manage_parser.set_defaults(func=manage_command)
+    manage_parser.add_argument('--django-help', action='store_true',
+                               help='Display the help for specific commands coming from Django. '
+                               'E.g. "tethys manage <django command> --django-help"')
+    manage_parser.set_defaults(func=manage_command, parsing_method='accepts_unknown_args')
 
 
-def manage_command(args):
+def manage_command(args, unknown_args=[]):
```

Comment:
Better to default to None, then check for None and assign as list in the function like so:
```
def manage_command(args, unknown_args=None):
    if not unknown_args:
        unknown_args = []
```